public/locales/de/credits.json

```diff
@@ -0,0 +1,9 @@
+{
+    "page-title": "Nachweise",
+    "creators": "Ersteller",
+    "developers": "Entwickler",
+    "ux-designers": "UX Designer",
+    "graphic-designers": "Grafikdesigner",
+    "partners": "Partner",
+    "tranlators":"translators"
```

Comment:
"translators" instead of `"tranlators"` (an "s" is missing)

Reply:
ok